sites/frontend.preview.zooniverse.org.conf

```diff
@@ -0,0 +1,132 @@
+server {
+    set $csp_whitelist "zooniverse.org *.zooniverse.org";
+    set $proxy_path "www.zooniverse.org/";
```

Comment:
Q? Do we want to match the production PFE system not the most recently version of PFE - if latest prod release then this works

Reply:
Production. The latest version is `master.pfe-preview.zooniverse.org` but that uses the staging API.